apps/web/screens/Organization/SubPage.tsx

```diff
@@ -68,7 +68,7 @@ const TOC: FC<React.PropsWithChildren<{ slices: Slice[]; title: string }>> = ({
           id: slice.id,
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore make web strict
-          text: slice['title'] ?? slice['leftTitle'] ?? '',
+          text: slice.json?.title ?? slice['title'] ?? slice['leftTitle'] ?? '',
```

Comment:
To keep this consistant we should either stick to the dot notation or accessing values via ['brackets']